trojan/Makefile

```diff
@@ -53,10 +53,10 @@ CMAKE_OPTIONS += \
 	-DENABLE_MYSQL=OFF \
 	-DENABLE_SSL_KEYLOG=ON \
 	-DENABLE_NAT=ON \
-	-DFORCE_TCP_FASTOPEN=OFF \
+	-DFORCE_TCP_FASTOPEN=ON \
```

Comment:
Do NOT try forcing its behavior

Reply:
This is when you compile trojan on a machine that doesn't support tfo but want tfo to work on another machine.